packages/apollo-gateway/src/index.ts

```diff
@@ -239,37 +245,50 @@ export class ApolloGateway implements GraphQLService {
     };
   }
 
-  protected async updateComposition(options?: {
```

Comment:
While I support the moving of the `options` (read: `engine`) from `updateComposition` to `load`, changing this from `protected` to `private` is a breaking change.  Are we bumping the major version of `@apollo/gateway`?

Reply:
I don't think this particularly matters & we could leave this `protected` if we're trying to avoid a breaking change.

Reply:
Removing the `options` from `updateComposition` is also part of what constitutes breaking, so maybe we just roll that back?

Reply:
Reverted these changes and added a comment for future consumption: 9a7e86c